configuration/override_dir_structure.rst

```diff
@@ -80,6 +82,22 @@ method::
 
 Here you have changed the location of the directory to ``app/{environment}/logs``.
 
+.. _override-templates-dir:
+
+Override the Templates Directory
+--------------------------------
+
+If your templates are not stored in the default ``app/Resources/views/``
+directory, use the :ref:`twig.paths <config-twig-paths>` configuration option to
+define your own templates directory (or directories):
```

Comment:
What about mentioning the root dir here? Either via a note or `%kernel.root_dir%/Resources/views/` instead of `app/Resources/views/`. This will be one of the things people will forget to change once they move their kernel to `src/`. People already are in the process of doing so now that the structure for 4.0 has been presented.